pkg/jetcloud/project.go

```diff
@@ -16,17 +16,25 @@ var ErrProjectAlreadyInitialized = errors.New("project already initialized")
 
 const dirName = ".jetpack.io"
 const configName = "project.json"
+const devConfigName = "dev.project.json"
 
 type projectConfig struct {
 	ProjectID id.ProjectID `json:"project_id"`
 	OrgID     id.OrgID     `json:"org_id"`
 }
 
-func InitProject(ctx context.Context, tok *session.Token, dir string) (id.ProjectID, error) {
+// JetCloud keeps configuration for the JetCloud API.
+// Could use a less stuttery name.
+type JetCloud struct {
```

Comment:
Nit, call this struct API?

And field can be Host

Reply:
Yeah, I considered `API` as well, but it feels odd for `API` to then be writing local files like `project.json`.

What about `jetcloud.Client`, and then we merge the other `client` from above?